### ISSUE TITLE
Checklist Step Creation not Working

### ISSUE THREAD

**Issue by githubbob42** [githubbob42/mingle2github2#570]
**[Mingle Card: 719](https://liquidframeworks.mingle.thoughtworks.com/projects/alpine_mobile/cards/719)**
QA Comment:
-----------

I had two checklists. One checklist (CL1) only had office value, the other checklist (CL2) had office, account and Job Type. I created a job that has all the attributes of CL2 and I then added a ticket on the job. Instead of picking CL2 the ticket picked CL1. I deleted the CL1 it still it did not pick CL2. Only after fixing the Ticket's record type data value to include underscore(i.e. from Field Ticket to Field\_Ticket), it picked the 'CL2' checklist.  The only record type on CL2 had the Job Type as Time\_Material.

I don't see the correlation between Ticket Type Record Type and Job Type Record Type.

Steps to reproduce
------------------

1.  Setup a basic checklist with a few steps
2.  Create a new ticket matching the checklist criteria

Expected result
---------------

The ticket should get the checklist steps

Actual Result
-------------

The new ticket does not get the checklist steps

### DEV Notes

Make sure that the JobType and TktType match exactly - including underscores.

The Job type might be "default" - make sure it is an option in the Picklist setup screen.

You shouldn't need to put in underscores anymore

Related Cards
-------------
